src/main/java/ca/team1310/swerve/odometry/FieldAwareSwerveDrive.java

```diff
@@ -109,6 +109,10 @@ public double getGyroYaw() {
         return gyro.getYaw();
     }
 
+    public double getGyroYawRate() {
```

Comment:
Make protected not public - not all gyros can give this data so if we can avoid adding it to the interface, let's do that.